service-base/src/main/java/org/eclipse/hono/service/management/device/DeviceManagementHttpEndpoint.java

```diff
@@ -37,7 +45,12 @@
  * Registration API</a>. It receives HTTP requests representing operation invocations and sends them to an address on
  * the vertx event bus for processing. The outcome is then returned to the peer in the HTTP response.
  */
-public final class DeviceManagementHttpEndpoint extends AbstractHttpEndpoint<ServiceConfigProperties> {
+public abstract class DeviceManagementHttpEndpoint extends AbstractHttpEndpoint<ServiceConfigProperties> {
```

Comment:
FMPOV the service-base module contains public API. As such, I think that we should not change this class but instead deprecate it do all modifications in AutowiredDeviceManagementHttpEndpoint only. WDYT?

Reply:
I reverted it and create a new AbstractDeviceManagementHttpEndpoint class instead